modules/oe_webtools_analytics/tests/src/Functional/ConfigurationTest.php

```diff
@@ -42,13 +45,13 @@ public function testLibraryLoading(): void {
 
     $this->drupalLogout();
     $this->drupalGet('<front>');
-    $this->assertSession()->responseContains('<script type="application/json">{"utility":"piwik","siteID":"123","sitePath":["ec.europa.eu"],"instance":"testing"}</script>');
+    $this->assertBodyContainsApplicationJson('{"utility":"piwik","siteID":"123","sitePath":["ec.europa.eu"],"instance":"testing"}');
```

Comment:
Here the front is the login page, which was accessed before, so if I dont add the event cache which contains the url.path context and the module settings tag, then this will fail here as it wont contain the json but what was cached in the previous login page call.